moped-database/migrations/1711054060504_add_component_work_types_tags_audit_fields/up.sql

```diff
@@ -0,0 +1,71 @@
+-- Creating or replacing a function to update audit logs in parent records
+CREATE OR REPLACE FUNCTION update_component_attributes_parent_records_audit_logs()
```

Comment:
I created this new function since the existing function that I was going to reuse called `update_parent_records_audit_logs` works on features records, like ones from `feature_drawn_lines`, that have the project component id stored as `component_id` instead of `project_component_id`.

That makes this function reusable for the future `moped_project_components_subcomponents` audit fields work. But, it leaves us with two very similar functions. What do y'all think?

Reply:
I think it makes total sense to create a new function, they are similar but different. Easier for me to follow than trying to abstract the other one imo.

Reply:
Going to make a more general function in the PR for https://github.com/cityofaustin/atd-data-tech/issues/16453. See https://austininnovation.slack.com/archives/CNUEPKLB1/p1711550047469799